assets/prefabs/behaviorNodes/findpathtonode.prefab

```diff
@@ -0,0 +1,11 @@
+{
+  "BehaviorNode": {
+    "type": "FindPathToNode",
+    "name": "Find Path",
+    "category": "movement",
+    "shape": "rect",
+    "description": "Finds a path to target using FlexiblePathfinding",
+    "color": [180, 180, 180, 255],
+    "textColor": [0, 0, 0, 255]
+  }
+}
```

Comment:
Should we just remove all of these prefabs and link them in a central issue to bring them back with support for the behavior tree editor? Having half of them might be more confusing than helpful...

Or are they actually complete (or only a few missing)?

Reply:
There still is the `ensureTargetPresent.prefab` which can be removed as we remove the node.
Apart from that we still only have the "old ones" for:
- condition
- jump
- log
- setTargetLocalPlayer
- setTargetToNearbyBlock

Would be nice if @DarkWeird could comment on my questions on the differences/changes between old originals and new replacements. Migrating the above-listed remaining ones in a similar fashion should be easy, but I'd like to understand the changes first before applying them to the rest.